modules/understanding-upgrade-channels.adoc

```diff
@@ -33,15 +60,25 @@ for more build information.
 [id="fast-version-channel_{context}"]
 == fast-{product-version} channel
 
-The `fast-{product-version}` channel is updated with new and previous minor versions of {product-version} as soon as Red Hat declares the given version as a general availability release. As such, these releases are fully supported, are production quality, and have performed well while available as a release candidate in the `candidate-{product-version}` channel from where they were promoted. Some time after a release appears in the `fast-{product-version}` channel, it is added to the `stable-{product-version}` channel. Releases never appear in the `stable-{product-version}` channel before they appear in the `fast-{product-version}` channel.
+The `fast-{product-version}` channel is updated with new and previous minor versions of {product-version} as soon as Red Hat declares the given version as a general availability (GA) release. The `fast-{product-version}` channel contains releases as soon as their errata are published.
+
+As such, these releases are fully supported and are production quality. The releases in this channel are closely monitored using a Telemetry system. The releases in the `fast-{product-version}` channel are promoted from `candidate-{product-version}` after the builds have performed well. Some time after a release appears in the `fast-{product-version}` channel, it is added to the `stable-{product-version}` channel.
```

Comment:
It was like this in the outgoing text too, but the discussion seems to be:

1. What fast is (GA releases, monitored, supported, etc.)
2. Where fast came from (candidate)
3. Where fast is going (stable)

If that's the breakdown, keeping at least the supported/monitored bits in the "what fast is" paragraph might help, and we can keep the latter two points for a separate paragraph, if it seems like to much to have all three points in one paraphaph.  As it stands, this incoming paragraph is mixing some of (1) with all of (2) and (3).